Resources/Prototypes/_EE/Catalog/uplink_catalog.yml

```
@@ -0,0 +1,40 @@
+- type: storeCategory
+  id: UplinkFaction
+  name: store-category-faction
+  priority: 12
+
+# Faction Gear Loadouts
+# Solarian Alliance
+- type: listing
+  id: UplinkFactionSolAllianceAsset
+  name: uplink-faction-san-asset-name
+  description: uplink-faction-san-asset-desc
+  icon: { sprite: /Textures/Structures/Storage/Crates/aluminiumcrate.rsi, state: icon }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)